### PR TITLE
Differentiated Arguments and ArgumentList properties

### LINKED ISSUE #2585: typo: ProcessStartInfo.Arguments and ArgumentList
**Submitted to [site feedback](https://github.com/MicrosoftDocs/feedback/issues/1601) by @ygoe**

Link to the documentation page:
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.arguments?view=netstandard-2.1

The following text probably has a typo:

In include quotation marks in the final parsed argument, triple-escape each mark.

Did you mean "To include quotation marks..."? Or is this a rare aspect of the English language I don't know about?

Link to the documentation page:
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.argumentlist?view=netstandard-2.1

There isn't much of an explanation why I would use ArgumentList over Arguments. Is there a platform-specific difference? The Arguments text only explains the situation on Windows. Is it different in Linux? Where does the string splitting happen? Or the list combination? (Whatever is used on the OS level.)

BTW: The content feedback box on the docs page is still missing.

### PR DESCRIPTION
**Pull request by rpetrusha**
## Differentiated Arguments and ArgumentList properties

Fixes #2585

### CODE REVIEW

File: xml/System.Diagnostics/ProcessStartInfo.xml [hunk 1]
```diff
@@ -237,7 +237,24 @@
         <summary>Gets a collection of command-line arguments to use when starting the application.</summary>
         <value>A collection of command-line arguments.</value>
         <remarks>
-          <format type="text/markdown"><![CDATA[If you use this property to set command-line arguments, <xref:System.Diagnostics.ProcessStartInfo.Arguments> must be set to an empty string.
+          <format type="text/markdown"><![CDATA[
+          
+`ArgumentList`, which is supported starting with .NET Core 2.1 and .NET Standard 2.1, and the <xref:System.Diagnostics.ProcessStartInfo.Arguments> property are independent of one another. That is, the strings assigned to the `ArgumentList` collection are not used to populate the <xref:System.Diagnostics.ProcessStartInfo.Argumenta> property, and the string assigned to the <xref:System.Diagnostics.ProcessStartInfo.Arguments> property is not parsed into individual strings that are assigned to `ArgumentList` collection.
```
Comment (BillWagner) [inline xml/System.Diagnostics/ProcessStartInfo.xml]: Typo:

```suggestion
`ArgumentList`, which is supported starting with .NET Core 2.1 and .NET Standard 2.1, and the <xref:System.Diagnostics.ProcessStartInfo.Arguments> property are independent of one another. That is, the strings assigned to the `ArgumentList` collection are not used to populate the <xref:System.Diagnostics.ProcessStartInfo.Arguments> property, and the string assigned to the <xref:System.Diagnostics.ProcessStartInfo.Arguments> property is not parsed into individual strings that are assigned to `ArgumentList` collection. 
```

File: xml/System.Diagnostics/ProcessStartInfo.xml [hunk 2]
```diff
@@ -322,14 +339,20 @@ info.ArgumentList.Add("-last Smith")
...
+        <value>A single string containing the arguments to pass to the target application specified in the <see cref="P:System.Diagnostics.ProcessStartInfo.FileName" /> property. The default is an empty string ("").   
+</value>
         <remarks>
-          <format type="text/markdown"><![CDATA[If you use this property to set command-line arguments, <xref:System.Diagnostics.ProcessStartInfo.ArgumentList*> must not contain any elements.
+          <format type="text/markdown"><![CDATA[
+
+The length of the string assigned to the `Arguments` property must be less than 32,699.
+  
+Arguments are parsed and interpreted by the target application, so must align with the expectations of that application. For.NET applications as demonstrated in the Examples below, spaces are interpreted as a separator between multiple arguments. A single argument that includes spaces must be surrounded by quotation marks, but those quotation marks are not carried through to the target application. To include quotation marks in the final parsed argument, triple-escape each mark.
```
Comment (BillWagner) [inline xml/System.Diagnostics/ProcessStartInfo.xml]: typo:

```suggestion
Arguments are parsed and interpreted by the target application, so must align with the expectations of that application. For .NET applications as demonstrated in the Examples below, spaces are interpreted as a separator between multiple arguments. A single argument that includes spaces must be surrounded by quotation marks, but those quotation marks are not carried through to the target application. To include quotation marks in the final parsed argument, triple-escape each mark.
```

File: xml/System.Diagnostics/ProcessStartInfo.xml [hunk 3]
```diff
@@ -322,14 +339,20 @@ info.ArgumentList.Add("-last Smith")
...
+If you use this property to set command-line arguments, <xref:System.Diagnostics.ProcessStartInfo.ArgumentList*> must not contain any elements.
```
Comment (BillWagner) [inline xml/System.Diagnostics/ProcessStartInfo.xml]: typo:

```suggestion
If you use this property to set command-line arguments, <xref:System.Diagnostics.ProcessStartInfo.ArgumentList> must not contain any elements.
```

**Comment by rpetrusha** [comment]
Thanks for catching these typos, @BillWagner!

**Pull request merged**